From 650b09131e33675475ba39401df20a0488112233 Mon Sep 17 00:00:00 2001
From: Jingchang Lu <jingchang.lu@freescale.com>
Date: Fri, 26 Sep 2014 16:21:48 +0800
Subject: [PATCH 020/128] Revert "arm: ls1021a: remove the FSL_SOC config
 added by SOC_LS1021A"

This reverts commit 14bbc976701a2ebc62343d3122e5ff772060a35f.

LS1021A shares IPs with sophisticated PowerPC platform,
many PowerPC drivers have depends on FSL_SOC defination,
so to consistent with this, FSL_SOC is introduced.

Signed-off-by: Jingchang Lu <jingchang.lu@freescale.com>
Change-Id: Ie5a69b78d317d09f9fee54dde3f1cd4bffdb9588
Reviewed-on: http://git.am.freescale.net:8181/19915
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Li Xiubo <Li.Xiubo@freescale.com>
Reviewed-by: Zhengxiong Jin <Jason.Jin@freescale.com>
[Xulin: Original patch taken from
QorIQ-SDK-V1.7-SOURCE-20141218-yocto_RDS_20150206.iso]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/mach-imx/Kconfig |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-imx/Kconfig b/arch/arm/mach-imx/Kconfig
index 11bec34..9a09c4f 100644
--- a/arch/arm/mach-imx/Kconfig
+++ b/arch/arm/mach-imx/Kconfig
@@ -829,6 +829,9 @@ config SOC_VF610
 	help
 	  This enable support for Freescale Vybrid VF610 processor.
 
+config FSL_SOC
+	bool
+
 config SOC_LS1021A
 	bool "Freescale LS1021A support"
 	select CPU_V7
@@ -836,6 +839,7 @@ config SOC_LS1021A
 	select CLKSRC_OF
 	select HAVE_ARM_ARCH_TIMER
 	select HAVE_SMP
+	select FSL_SOC
 	select MIGHT_HAVE_PCI
 	select PCI_DOMAINS if PCI
 	select ZONE_DMA if ARM_LPAE
-- 
1.7.5.4

